node/core/av-store/src/lib.rs

```diff
@@ -564,6 +578,10 @@ async fn run_iteration<Context>(
 	ctx: &mut Context,
 	subsystem: &mut AvailabilityStoreSubsystem,
 	mut next_pruning: &mut future::Fuse<Delay>,
+	(pruning_result_tx, pruning_result_rx): (
+		&mut MpscSender<Result<(), Error>>,
+		&mut MpscReceiver<Result<(), Error>>,
+	),
```

Comment:
nit: It would be simpler for this not to be a tuple, and it would save two lines. 😛

Reply:
you mean just to have two separate params pruning_result_tx and pruning_result_rx ?

Reply:
Yep. Just a style nit, feel free to ignore if you want.

Reply:
I will keep it as it is than :) , since I think it makes sense to try to have closely related params in the same data structure.